com.github.lruzicka.Needly.yaml

```diff
@@ -0,0 +1,113 @@
+# The YAML manifest for the Needle Flatpak
+#
+app-id: com.github.lruzicka.Needly
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: needly
+finish-args: 
+   - --share=ipc # For X11
+   - --socket=wayland # Enable communication with Wayland
+   - --socket=x11 # Alternatively, with X11
+   - --device=kvm # Allow access to a KVM device
+   - --filesystem=home # Allow read-write access to home dir
+   - --filesystem=host # Allow read-write access to host dir tree
+   - --filesystem=xdg-run/gvfs # Access gvfs mounts
+   - --filesystem=xdg-run/libvirt # Access to qemu:///session
+   - --filesystem=/run/libvirt # Access to qemu:///system
+
+
+# Installing the application and the dependencies
+#
+modules:
+  # This will install the Tkinter support for Python as
+  # Tkinter is not part of the runtime.
+  - python3-tkinter.yaml
+  # This installs the Pillow library needed to work with images.
+  # It uses PIP.
+  - name: pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps .
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/03/a3/f61a9a7ff7969cdef2a6e0383a346eb327495d20d25a2de5a088dbb543a6/Pillow-9.0.1.tar.gz
+        sha256: 6c8bc8238a7dfdaf7a75f5ec5a663f4173f8c367e5a39f87e720495e1eed75fa
+
+  # This is a dependency for libvirt.
+  - name: libtirpc
+    config-opts:
+      - --disable-gssapi
+      - --libdir=/app/lib
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/sourceforge/libtirpc/libtirpc-1.3.2.tar.bz2
+        sha256: e24eb88b8ce7db3b7ca6eb80115dd1284abc5ec32a8deccfed2224fc2532b9fd
+
+  # This is another dependency for libvirt which is used to
+  # create the necessary documentation.
+  - name: docutils
```

Comment:
can't you just disable building libvirt documentation?

Reply:
On my local build, Libvirt fails to build without documentation.

Reply:
you mean passing `-Ddocs=false` as a config opt to libvirt doesn't work?

Reply:
> you mean passing `-Ddocs=false` as a config opt to libvirt doesn't work?

When doing `-Ddocs=false` the compilation complains about the keyword and suggest either `enable`, `disable`, or `auto`. I tried with `disable` and the documentation did not compile, but more and more errors started to appear, so I let it build and with it it works normally.